### PR TITLE
New text for Why Ember-CLI

### PR DESCRIPTION
**Pull request by sandstrom**
While the existing text was very useful in 2014 when it was posted few newcomers know about ember-app-kit and I think everyone that was using it know about the benefits of Ember-CLI.

I've repurposed the section to explain why a shared tooling infrastructure is useful.

Thoughts?

### CODE REVIEW

**Comment by stefanpenner** [comment]
Ya, it is totally time to reword this. I am not the best person to review this, but so far what i see looks good/better.

@locks could use your 👀  here.

File: _posts/2014-04-04-why.md [hunk 1]
```diff
@@ -5,16 +5,11 @@ permalink: why
 github: "https://github.com/ember-cli/ember-cli.github.io/blob/master/_posts/2014-04-04-why.md"
 ---
 
-The [Ember App Kit](https://github.com/stefanpenner/ember-app-kit) (_now deprecated_) project proved to be quite useful.
-We've learned a lot, and it allowed us to iterate quickly while building real ambitious applications.
+A good browser framework is important. But building ambitious web applications also require excellent tooling and process:
```
Comment (locks) [inline _posts/2014-04-04-why.md]: Ember CLI

**Comment by sandstrom** [comment]
@locks @stefanpenner I've included locks changes.

Do you want a squash or can you use the squash + merge feature in Github?

**Comment by nathanhammond** [comment]
I don't mind leaving the `git` history. If @locks is cozy from the learning team's perspective I'm in favor of shipping. The content seems good and I don't care to over-wordsmith it.

**Comment by locks** [comment]
> The content seems good and I don't care to over-wordsmith it.

This. :shipit:

**Pull request merged**